readthedocs/projects/views/private.py

```diff
@@ -154,6 +156,15 @@ def project_version_detail(request, project_slug, version_slug):
     if request.method == 'POST' and form.is_valid():
         version = form.save()
         if form.has_changed():
+            if 'slug' in form.changed_data and version.slug not in (STABLE, LATEST):
+                # Rebuild symlinks to the new slug, remove the old slug's symlink
+                # Latest and Stable would appear here because they are disabled on the form
+                log.info('Rebuilding symlinks for %s. A version slug changed', version.project)
+                broadcast(type='app', task=tasks.symlink_project, args=[version.project.pk])
+
+                log.info('Triggering a build of the moved version')
+                trigger_build(version.project, version)
+
             if 'active' in form.changed_data and version.active is False:
```

Comment:
What would happen if we `de-active` a version and change the `slug` in the same request?

Since this is triggering async task something weird may happen. Not sure yet, though.

Reply:
I'll check this edge case.